install_radar_deps_opensuse.sh

```diff
@@ -134,16 +134,31 @@ zypper install -y kernel-devel
 wget http://developer.download.nvidia.com/compute/cuda/10.1/Prod/local_installers/cuda_10.1.243_418.87.00_linux.run
 sh cuda_10.1.243_418.87.00_linux.run --silent --toolkit --samples 
 
+### INSTALL PYDARN FOR REALTIME AND TESTING ###
+cd $HOME
+git clone https://github.com/SuperDARN/pydarn.git 
+
 #### REALTIME ####
 cd /usr/local
 git clone https://github.com/vtsuperdarn/hdw.dat.git
-cd -
 mkdir $BOREALISPATH/borealisrt_env
 virtualenv $BOREALISPATH/borealisrt_env
 source $BOREALISPATH/borealisrt_env/bin/activate
 pip install zmq
 pip install git+git://github.com/SuperDARNCanada/backscatter.git#egg=backscatter
-git clone https://github.com/SuperDARN/pydarn.git
-cd pydarn
+cd $HOME/pydarn
 git checkout develop
 python setup.py install
+deactivate
```

Comment:
Just checking that you want the pydarn setup.py install before the deactivation of the borealisrst_env virtualenv?
I'm not too familiar with virtualenvs so don't know if this has any side effects.

Reply:
Yes this is as intended. Installing from the develop branch on this borealisrt_env virtualenv, then deactivating to go to the other virtualenv pydarn-env. Pydarn will remain installed on the virtualenv for any processes that use that virtualenv.